### DISCUSSION TITLE
Question about object sorting when sequential printing

### DISCUSSION THREAD

**Discussion by pimmen85** [supermerill/SuperSlicer#3682] [General]
sometimes its effective to print several smaller object at once, but i cant get it to work effortless, yes i can fill the whole bed, and then choose "from right panel" as sorting, but that requires lots of manual work and aint really logical, even if i drag an object to the top to print it forst, it doesnt end up there, it would be nice to have options that said like "print from back of bed" or "print from left to right" etcetera.

am i the only one with these issues?

**Comment by neophyl** [comment]
It does.  Sort of.  Under Print Settings>Sequential Printing>Object sort, there are options for Right Panel, Lowest Z and Lowest Y.  When lowest Y is selected it prints from the front of your plater to the back (for a plater with origin front left at least).  Just arrange in staggered lines

**Reply by pimmen85** [reply]
yea, but my problem is i would like to be able to print from "the back" because i have a aftermarket printed fan setup so it sticks out quite a bit more than the stock one.
